### PR TITLE
Added support for net6.0

### PR DESCRIPTION
**Pull request by quigamdev**
fixed loading extension methods dependent on System.Linq assembly

### CODE REVIEW

File: src/Samples/Api.AspNetCoreLatest/global.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (acizmarik) [inline src/Samples/Api.AspNetCoreLatest/global.json:1]: Please remove this file

File: src/Samples/AspNetCoreLatest/global.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (acizmarik) [inline src/Samples/AspNetCoreLatest/global.json:1]: Please remove this as well

**Review by acizmarik — APPROVED** [review]

**Comment by acizmarik** [comment]
Also please increase version for github actions

**Comment by quigamdev** [comment]
@acizmarik  I had to fix tests that are dependent on .net5.0. Please check the PR again 🙏🏻

File: src/Tests/ViewModel/ViewModelTypeMetadataSerializerTests.cs [hunk 3]
```diff
@@ -33,19 +33,19 @@ public static void ClassInit(TestContext context)
...
+        [DataRow(typeof(IDictionary<int, int>), "[\"COeXIi+/gCnEgi6U\"]")]
+        [DataRow(typeof(Dictionary<object, object>), "[\"y6hj7iGE5x+eLypR\"]")]
+        [DataRow(typeof(IDictionary<object, object>), "[\"y6hj7iGE5x+eLypR\"]")]
+        [DataRow(typeof(List<KeyValuePair<string, string>>), "[\"PVyTyqvI4Id5nYo+\"]")]
+        [DataRow(typeof(List<KeyValuePair<int, int>>), "[\"COeXIi+/gCnEgi6U\"]")]
+        [DataRow(typeof(List<KeyValuePair<object, object>>), "[\"y6hj7iGE5x+eLypR\"]")]
+        [DataRow(typeof(IList<KeyValuePair<string, string>>), "[\"PVyTyqvI4Id5nYo+\"]")]
+        [DataRow(typeof(IList<KeyValuePair<int, int>>), "[\"COeXIi+/gCnEgi6U\"]")]
+        [DataRow(typeof(IList<KeyValuePair<object, object>>), "[\"y6hj7iGE5x+eLypR\"]")]
         // these hashes are dependent on the target framework - the latest update of hashes is updated to net50
```
Comment (acizmarik) [inline src/Tests/ViewModel/ViewModelTypeMetadataSerializerTests.cs:49]: This comment should be updated to reflect that these hashes are updated for net60

**Review by acizmarik — APPROVED** [review]

**Pull request merged**